templates/data-providers/index.jsx

```diff
@@ -33,7 +33,20 @@ const SearchResults = ({
     return (
       <Search.Results>
         <Search.Result>
-          <b>No results found</b>
+          <h2>
+            Is there a mistake?{' '}
+            <span role="img" aria-label="">
```

Comment:
I didn't use any label because Chrome's screen reader takes a big idle period between sentence and emoji label.

Reply:
Keep it presentational only then. Maybe with `aria-hidden`, but I am not sure in this case.

Reply:
I believe aria-hidden="true" and aria-label="" are interchangeable, aren't they. I put it like this because of eslint rule